### ISSUE TITLE
The recommended path for the REQUESTS_CA_BUNDLE environment variable is incorrect for 64-bit Windows

### ISSUE THREAD

**Issue by sethlivingston** [MicrosoftDocs/azure-docs-cli#4055]
### Prerequisites

- [X] Write a descriptive title.
- [X] Search the existing issues.
- [X] I am reporting the documentation problem for the current version of Azure CLI.

### Azure CLI version being used

2.55.0

### Link to affected document

https://learn.microsoft.com/en-us/cli/azure/use-cli-effectively?tabs=bash%2Cbash2#work-behind-a-proxy

### Description of the documentation error

The recommended path for the REQUESTS_CA_BUNDLE on Windows is
```
C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\Lib\site-packages\certifi\cacert.pem
```
But on 64-bit Windows it's in
```
C:\Program Files\...
```

### Suggested fix

It would be nice if we could just say `%ProgramFiles%\...`, but that environment variable isn't set consistently across CMD and different versions of Powershell. I would probably list them both?

**Comment by SaibabaBalapur-MSFT** [comment]
@sethlivingston 
Thanks for bringing this to our attention.
I'm going to assign this to the document author so they can take a look at it accordingly.

**Comment by dbradish-microsoft** [comment]
@sethlivingston, Thank you for bringing this to our attention.  This has been fixed in #4075
